lms/templates/forgot_password_modal.html

```diff
@@ -30,7 +30,7 @@ <h5 class="modal-title  f-18 h-35" id="forgot-password-content">${_("Forgot pass
 
                           <div class="field required text form-group" id="forgot-password-modal-field-email">
                               <label for="pwd_reset_email" class="f-16 h-26">${_("E-mail")}</label>
-                              <input class="js-email-forgot-password-field" id="pwd_reset_email" type="email" name="email" value="" aria-label='${_("Enter your E-mail Address to reset your password")}'  aria-describedby="pwd_reset_email-tip" aria-required="true" />
+                              <input class="js-email-forgot-password-field" id="pwd_reset_email" type="email" name="email" value="" aria-label='${_("Enter your E-mail Address to reset your password")}' aria-required="true" />
```

Comment:
Przeoczyłam to  wcześniej, ale aria-label powinno zaczynać się od tego samego tekstu jaki jest wizualnie (wywoływanie głosem) dlatego proszę  ododanie E-mail na początku

Reply:
Dodałam